### PR TITLE
Initial openapi.yaml, standardize terms and naming

### PR DESCRIPTION
**Pull request by iridescent-beacon**

### CODE REVIEW

**Review by artem30801 — APPROVED** [review]

File: openapi.yaml [hunk 1]
```diff
@@ -0,0 +1,254 @@
...
+              schema:
+                $ref: "#/components/schemas/EmptyObject"
+        '400':
+          description: Bad request.
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/Error"
+        '403':
+          description: Forbidden.
```
Comment (ypoluektovich) [inline openapi.yaml:73]: We'll probably want a bit more verbose descriptions for errors at some point. E.g. here, why could the request be forbidden?

File: openapi.yaml [hunk 2]
```diff
@@ -0,0 +1,254 @@
...
+        - sessions
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/LogInRequest'
+      responses:
+        '200':
+          description: Success.
```
Comment (ypoluektovich) [inline openapi.yaml]: Probably should mention that the session token is returned in the cookie

File: openapi.yaml [hunk 3]
```diff
@@ -0,0 +1,254 @@
...
+        '400':
+          description: Bad request.
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/Error"
+  /sessions:
+    post:
+      summary: Log in to an existing account.
+      operationId: logIn
```
Comment (ypoluektovich) [inline openapi.yaml]: Should we make operation ids match the method names in the code?

**Review by ypoluektovich — APPROVED** [review]

File: openapi.yaml (hunk 1 above)
Comment (iridescent-beacon) [reply openapi.yaml:73]: Definitely! They all currently use the same example too, so they all show `bad request` which I'm not sure how best to address offhand.

File: openapi.yaml (hunk 3 above)
Comment (iridescent-beacon) [reply openapi.yaml]: How do you feel about settling on
* `create_account`
* `create_session`
* `get_signals`
* `patch_signals`

Trying to remember that the "accounts" url is maps to "user" in the backend and that "log_in" is spelled "session" In the front end isn't stellar.

I'm not in love with the `/signals` endpoint returning a struct named `Tags` either, particularly as in a later commit I add a `/tags` endpoint which I spelled `JustTags` for expediency at the time :p 

If there aren't objections I'll rename the `"user"` table to `account` while I'm at it so we can stop quoting it everywhere.

Comment (ypoluektovich) [reply openapi.yaml]: Yeah, making the names be consistent across the project sounds like a good idea.

File: openapi.yaml [hunk 4]
```diff
@@ -45,6 +45,11 @@ paths:
       responses:
         '200':
           description: Success.
+          headers:
+            FicAiSession:
```
Comment (ypoluektovich) [inline openapi.yaml]: `FicAiSession` is not the name of a header, it's the name of a cookie

File: openapi.yaml (hunk 3 above)
Comment (iridescent-beacon) [reply openapi.yaml]: Gave it a pass to cleanup naming and rearrange a few things, please let me know if there's anything outstanding you see.

File: openapi.yaml (hunk 4 above)
Comment (iridescent-beacon) [reply openapi.yaml]: Oops, thanks; fixed and added to create session

File: openapi.yaml [hunk 5]
```diff
@@ -0,0 +1,268 @@
...
+        error:
+          type: object
+          required:
+            - message
+          properties:
+            message:
+              description: Human readable error message.
+              type: string
+              example: 'bad request'
+    CreateAccountRequest:
```
Comment (ypoluektovich) [inline openapi.yaml]: It's called `CreateAccountQ` in server code, maybe use the same name here?

Comment (iridescent-beacon) [reply openapi.yaml]: Changed.

**Review by ypoluektovich — APPROVED** [review]

**Pull request merged**